### PR TITLE
update README.md

### PR DESCRIPTION
**Pull request by cleverhu**

### CODE REVIEW

**Comment by Iceber** [comment]
@windsonsea PTAL, Thanks

File: README.md [hunk 1]
```diff
@@ -10,6 +10,42 @@ Clusterpedia can synchronize resources with multiple clusters and provide more p
...
+* Support for retrieving resources using `kubectl`, `client-go` or `controller-runtime/client`, [client-go example](https://github.com/clusterpedia-io/client-go/blob/main/examples/list-clusterpedia-resources/main.go)
+* The metadata of the resource an be retrived via API or [client-go/metadata](https://pkg.go.dev/k8s.io/client-go/metadata)
+* Rich retrieval conditions: [Filter by cluster/namespace/name/creation](https://clusterpedia.io/docs/usage/search/multi-cluster/#basic-features), [Search by parent or ancestor owner](https://clusterpedia.io/docs/usage/search/multi-cluster/#search-by-parent-or-ancestor-owner),[Multi-Cluster Label Selector](https://clusterpedia.io/docs/usage/search/#label-selector), [Enhanced Field Selector](https://clusterpedia.io/docs/usage/search/#field-selector), [Custom Search Conditions](https://clusterpedia.io/docs/usage/search/#advanced-searchcustom-conditional-search), etc.
+### Support for importing 1.10+ versions of Kubernetes
+### Automic conversion of different versions of Kube resources and support for multiple version of resources
+* Even if you import different version of Kube, we can still use the same resource version to retrieve resources
```
Comment (windsonsea) [inline README.md:23]: Even if you use different Kubernetes versions, you can still retrieve resources with a same version through Clusterpedia.

File: README.md [hunk 2]
```diff
@@ -10,6 +10,42 @@ Clusterpedia can synchronize resources with multiple clusters and provide more p
...
+> For example, we can use `v1`, `v1beta2`, `v1beta1` version to retrieve the Deployments resources in different clusters.
```
Comment (windsonsea) [inline README.md:24]: For example, you can use a version number such as `v1`, `v1beta2`, or `v1beta1` to retrieve deployments running in different clusters.

File: README.md [hunk 3]
```diff
@@ -10,6 +10,42 @@ Clusterpedia can synchronize resources with multiple clusters and provide more p
...
+> 
+>   Notes: In 1.10 *deployments* is `v1beta1`, in 1.24 *deployments* is `v1`
```
Comment (windsonsea) [inline README.md:26]: Notes: The version of *deployments* is `v1beta1` in In Kubernetes 1.10 and it is `v1` in Kubernetes 1.24.

File: README.md [hunk 4]
```diff
@@ -10,6 +10,42 @@ Clusterpedia can synchronize resources with multiple clusters and provide more p
...
+### A single API can be used to retrieve different types of resources
```
Comment (windsonsea) [inline README.md:27]: Retrieve different workloads through a single API

File: README.md [hunk 5]
```diff
@@ -10,6 +10,42 @@ Clusterpedia can synchronize resources with multiple clusters and provide more p
...
+* Use [`Collection Resource`](https://clusterpedia.io/docs/concepts/collection-resource/) to retrieve different types of resources, such as `Deployment`, `DaemonSet`, `StatefulSet`.
```
Comment (windsonsea) [inline README.md:28]: Use [`Collection Resource`](https://clusterpedia.io/docs/concepts/collection-resource/) to retrieve different workloads such as `Deployment`, `DaemonSet`, `StatefulSet`.

File: README.md [hunk 6]
```diff
@@ -10,6 +10,42 @@ Clusterpedia can synchronize resources with multiple clusters and provide more p
...
+```bash
+$ kubectl get collectionresources
+NAME            RESOURCES
+workloads       deployments.apps,daemonsets.apps,statefulsets.apps
+kuberesources   *,*.admission.k8s.io,*.admissionregistration.k8s.io,*.apiextensions.k8s.io,*.apps,*.authentication.k8s.io,*.authorization.k8s.io,*.autoscaling,*.batch,*.certificates.k8s.io,*.coordination.k8s.io,*.discovery.k8s.io,*.events.k8s.io,*.extensions,*.flowcontrol.apiserver.k8s.io,*.imagepolicy.k8s.io,*.internal.apiserver.k8s.io,*.networking.k8s.io,*.node.k8s.io,*.policy,*.rbac.authorization.k8s.io,*.scheduling.k8s.io,*.storage.k8s.io
+```
+### Diverse resource sync policies and intelligent sync of cutom resources with AA resources.
```
Comment (windsonsea) [inline README.md:35]: Diverse policies and intelligent synchronization

File: README.md [hunk 7]
```diff
@@ -10,6 +10,42 @@ Clusterpedia can synchronize resources with multiple clusters and provide more p
...
+* [Wildcards](https://clusterpedia.io/docs/usage/sync-resources/#using-wildcards-to-sync-resources) can be used to sync all types of resources within a specified group or cluster.
+* [Support for synchronizing all custom resources](https://clusterpedia.io/docs/usage/sync-resources/#sync-all-custom-resources)
+* The type of resources to be synchronized can be automatically adjusted accoriding to CRD and AA changes
```
Comment (windsonsea) [inline README.md:38]: The resource types you are synchronizing can be adapted to your CRD and AA changes.

File: README.md [hunk 8]
```diff
@@ -10,6 +10,42 @@ Clusterpedia can synchronize resources with multiple clusters and provide more p
...
+### Unify the search entry for master clusters and multi-cluster resources
+* Based on [Aggregated API](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/apiserver-aggregation/), the entry portal for multi-cluster retrieval is the same as that of the the master cluster(IP:PORT)
+### Very low memory usage and weak network optimization
+* Optimize informer's cache store for low memory usage for resource synchronization
```
Comment (windsonsea) [inline README.md:42]: Optimized caches used by informer, so the memory usage is very low for resource synchronization

File: README.md [hunk 9]
```diff
@@ -10,6 +10,42 @@ Clusterpedia can synchronize resources with multiple clusters and provide more p
...
+* Automatic start/stop synchronization based on cluster health status
+### High availability
+### No dependency on specific storage components
+Clusterpedia does not care about storage components and uses storage layer to interface to specific storage components,
```
Comment (windsonsea) [inline README.md:46]: Clusterpedia does not care about storage components and uses the storage layer to attach specific storage components,

**Review by windsonsea — COMMENTED** [review]
some text optimization

File: README.md [hunk 10]
```diff
@@ -79,6 +94,15 @@ resources               clusterpedia.io/v1beta1  false   Resources
 More information about [Search Conditions](https://clusterpedia.io/docs/usage/search/),
 about [Label Selector](https://clusterpedia.io/docs/usage/search/#label-selector) and [Field Selector](https://clusterpedia.io/docs/usage/search/#field-selector)
 
+## Usage Samples
+We can search for resources configured in *PediaCluster*, Clusterpedia supports two types of resource search:
```
Comment (windsonsea) [inline README.md:98]: You can search for resources configured in *PediaCluster*. Clusterpedia supports two resource types:

File: README.md [hunk 11]
```diff
@@ -79,6 +94,15 @@ resources               clusterpedia.io/v1beta1  false   Resources
...
+* Search for resources that are compatible with Kubernetes OpenAPI
+* Search for [`Collection Resource`](https://clusterpedia.io/docs/concepts/collection-resource/)
+```sh
+$ kubectl api-resources | grep clusterpedia.io
+collectionresources     clusterpedia.io/v1beta1  false   CollectionResource
+resources               clusterpedia.io/v1beta1  false   Resources
+```
 ### Search in Kubernetes OpenAPI Compatible Way
```
Comment (windsonsea) [inline README.md:106]: Use a  compatible way with Kubernetes OpenAPI

File: README.md [hunk 12]
```diff
@@ -79,6 +94,15 @@ resources               clusterpedia.io/v1beta1  false   Resources
 More information about [Search Conditions](https://clusterpedia.io/docs/usage/search/),
 about [Label Selector](https://clusterpedia.io/docs/usage/search/#label-selector) and [Field Selector](https://clusterpedia.io/docs/usage/search/#field-selector)
 
+## Usage Samples
+We can search for resources configured in *PediaCluster*, Clusterpedia supports two types of resource search:
+* Search for resources that are compatible with Kubernetes OpenAPI
```
Comment (windsonsea) [inline README.md:99]: * Resources that are compatible with Kubernetes OpenAPI

File: README.md [hunk 13]
```diff
@@ -79,6 +94,15 @@ resources               clusterpedia.io/v1beta1  false   Resources
...
+* Search for [`Collection Resource`](https://clusterpedia.io/docs/concepts/collection-resource/)
```
Comment (windsonsea) [inline README.md:100]: * [`Collection Resource`](https://clusterpedia.io/docs/concepts/collection-resource/)

File: README.md [hunk 14]
```diff
@@ -79,6 +94,15 @@ resources               clusterpedia.io/v1beta1  false   Resources
...
+```sh
+$ kubectl api-resources | grep clusterpedia.io
+collectionresources     clusterpedia.io/v1beta1  false   CollectionResource
+resources               clusterpedia.io/v1beta1  false   Resources
+```
 ### Search in Kubernetes OpenAPI Compatible Way
 Although we can use URL to search resources, if we want to use `kubectl` to query more conveniently,
```
Comment (windsonsea) [inline README.md:107]: It is possible to search resources via URL, but using `kubectl` may be more convenient if

File: README.md [hunk 15]
```diff
@@ -79,6 +94,15 @@ resources               clusterpedia.io/v1beta1  false   Resources
...
+* Search for resources that are compatible with Kubernetes OpenAPI
+* Search for [`Collection Resource`](https://clusterpedia.io/docs/concepts/collection-resource/)
+```sh
+$ kubectl api-resources | grep clusterpedia.io
+collectionresources     clusterpedia.io/v1beta1  false   CollectionResource
+resources               clusterpedia.io/v1beta1  false   Resources
+```
 ### Search in Kubernetes OpenAPI Compatible Way
 Although we can use URL to search resources, if we want to use `kubectl` to query more conveniently, 
 we need to [configure the cluster shortcut for `kubectl`](https://clusterpedia.io/docs/usage/access-clusterpedia/#configure-the-cluster-shortcut-for-kubectl).
```
Comment (windsonsea) [inline README.md:108]: you [configured shortcuts for `kubectl`](https://clusterpedia.io/docs/usage/access-clusterpedia/#configure-the-cluster-shortcut-for-kubectl).

**Review by windsonsea — COMMENTED** [review]
More comments come with two main points:
1. Use `you` instead of `we` because `you` are users and `we` are developers
2. Keep consistency in all titles, uppercases, punctuations

File: README.md [hunk 16]
```diff
@@ -79,6 +94,15 @@ resources               clusterpedia.io/v1beta1  false   Resources
 More information about [Search Conditions](https://clusterpedia.io/docs/usage/search/),
 about [Label Selector](https://clusterpedia.io/docs/usage/search/#label-selector) and [Field Selector](https://clusterpedia.io/docs/usage/search/#field-selector)
```
Comment (windsonsea) [inline README.md:95]: [Label Selector](https://clusterpedia.io/docs/usage/search/#label-selector) and [Field Selector](https://clusterpedia.io/docs/usage/search/#field-selector)

**Review by Iceber — APPROVED** [review]
LGTM

**Pull request merged**